test/distributed/org/apache/cassandra/distributed/test/accord/AccordMigrationTest.java

```diff
@@ -696,6 +705,27 @@ private static void assertMigrationState(String tableName, ConsensusMigrationTar
         });
     }
 
+    private static void assertVtableMigrationState(String tableName, List<Range<Token>> migratedRanges, List<Range<Token>> migratingRanges) throws JsonProcessingException
+    {
+        SimpleQueryResult vtableResult =
+            SHARED_CLUSTER.get(1).executeInternalWithResult("SELECT * FROM system_views.accord_migration_state WHERE table_name = ?", tableName);
+        assertTrue(vtableResult.hasNext());
+        Row tableState = vtableResult.next();
+        List<String> vtableMigratedRanges = tableState.getList("migrated_ranges");
+        assertEquals(migratedRanges, vtableMigratedRanges.stream().map(Range::fromString).collect(Collectors.toList()));
```

Comment:
@aweisberg It looks like this assertion is a bit flakey. I haven't been able to reproduce locally, but a couple times in CI...

```
junit.framework.AssertionFailedError: expected:<[]>  but was:<[(-1,9223372036854775807]]>
	at org.apache.cassandra.distributed.test.accord.AccordMigrationTest.assertVtableMigrationState(AccordMigrationTest.java:715)
	at org.apache.cassandra.distributed.test.accord.AccordMigrationTest.assertMigrationState(AccordMigrationTest.java:633)
	at org.apache.cassandra.distributed.test.accord.AccordMigrationTest.lambda$testPaxosToAccordCAS$13(AccordMigrationTest.java:404)
```

Reply:
I'm not sure how that range has moved to "migrated", unless I'm looking at the wrong keyspace/table...or there was something I'm waiting for that I shouldn't (as this looks a lot like I'm getting a later state than expected)